core/store/orm/config_reader.go

```diff
@@ -60,6 +60,7 @@ type ConfigReader interface {
 	GasUpdaterBlockDelay() uint16
 	GasUpdaterBlockHistorySize() uint16
 	GasUpdaterTransactionPercentile() uint16
+	InsecureSkipVerify() bool
```

Comment:
If it's only used in the client does it make sense to put it in the main config struct? Perhaps reading directly from ENV might be better, wdyt?

Reply:
I'd prefer not to diverge from convention just for this CLI var, when there are other CLI vars in the config struct with the same issue, this would also stop the config var from loading from a config file which we still support.